frontend/pages/create-project.tsx

```diff
@@ -121,21 +171,24 @@ const CreateProject: React.FC = () => {
 
                     <div className="flex flex-col md:flex-row items-start mt-10">
                         <div className="col-span-full mb-4 md:mb-0">
-                            <label htmlFor="about" className="block text-sm font-medium leading-6 text-gray-900">
-                                About the project
+                            <label htmlFor="description" className="block text-sm font-medium leading-6 text-gray-900">
+                                Description
                             </label>
                             <div className="mt-2">
                                 <textarea
-                                    id="about"
-                                    name="aboutProject"
+                                    id="description"
+                                    name="description"
                                     rows={10}
                                     maxLength={projectDescriptionMaxLength}
-                                    className="p-1 block w-full md:w-auto rounded-md border-0 py-1.5 text-gray-900 shadow-sm ring-1 ring-inset ring-gray-300 placeholder:text-gray-400 focus:ring-2 focus:ring-inset focus:ring-indigo-600 md:text-sm md:leading-6"
-                                    value={formik.values.aboutProject}
+                                    className="form-box"
+                                    value={formik.values.description}
                                     onChange={formik.handleChange}
                                 />
                             </div>
-                            <p className="text-sm text-gray-500">{`${formik.values.aboutProject.length}/${projectDescriptionMaxLength} characters`}</p>
+                            <p className="text-sm text-gray-500">{`${formik.values.description.length}/${projectDescriptionMaxLength} characters`}</p>
+                            {formik.touched.description && formik.errors.description ? (
```

Comment:
Don't use ternary operator to return null if false :
{condition && (
  <Code />
)}